### PR TITLE
Update concepts.md

### PR DESCRIPTION
**Pull request by munukutla**
Fixed grammar

### CODE REVIEW

**Comment by munukutla** [comment]
I've signed it.

File: docs/guides/concepts.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ message HelloResponse {
 ```
 
 
-gRPC lets you define four kinds of service method:
+gRPC lets you define four kinds of service methods:
```
Comment (LisaFC) [inline docs/guides/concepts.md]: The original version here ("kinds of method") is correct.

File: docs/guides/concepts.md [hunk 2]
```diff
@@ -65,7 +65,7 @@ rpc LotsOfReplies(HelloRequest) returns (stream HelloResponse){
 - Client streaming RPCs where the client writes a sequence of messages and sends
   them to the server, again using a provided stream. Once the client has
   finished writing the messages, it waits for the server to read them and return
-  its response.
+  a single response.
```
Comment (LisaFC) [inline docs/guides/concepts.md]: a single -> its single

**Comment by thisisnotapril** [comment]
Closing due to age, no LF CLA, changes still needed review

**Comment by thelinuxfoundation** [comment]
Thank you for your pull request. Before we can look at your contribution, we need to ensure all contributors are covered by a Contributor License Agreement.

After the following items are addressed, please respond with a new comment here, and the automated system will re-verify.

* User @munukutla isn't covered by a CLA. They will need to complete the form at https://identity.linuxfoundation.org/projects/cncf

_Regards,
CLA GitHub bot_

**Pull request closed without merging**